src/InputStepper/index.js

```diff
@@ -0,0 +1,331 @@
+  border: none;
+  padding: 0 0 ${getSizeToken(TOKENS.PADDINGINPUT)};
+  font-size: inherit;
+  color: inherit;
+  background-color: inherit;
+  cursor: inherit;
+  width: 100%;
+  height: 100%;
+  flex: 1 auto;
+  text-align: center;
+  border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
+  z-index: 2;
+
+  &:focus {
+    outline: none;
+    & ~ ${FakeInput} {
+      box-shadow: inset 0 0 0
+        ${({ theme }) =>
+          `${theme.orbit.borderWidthInputFocus} ${theme.orbit.borderColorInputFocus}`};
+    }
+  }
+
+  &::placeholder {
+    color: ${({ theme }) => theme.orbit.colorPlaceholderInput};
+    /* Firefox */
+    opacity: 1;
+  }
+
+  /* Internet Explorer 10-11 */
+  &:-ms-input-placeholder {
+    color: ${({ theme }) => theme.orbit.colorPlaceholderInput};
+  }
+
+  /* Microsoft Edge */
+  &::-ms-input-placeholder {
+    color: ${({ theme }) => theme.orbit.colorPlaceholderInput};
+  }
+  &::-webkit-inner-spin-button,
+  ::-webkit-outer-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
+`;
+
+Input.defaultProps = {
+  theme: defaultTokens,
+};
+
+const FormLabel = ({
+  label,
+  isFilled,
+  required,
+}: {
+  label?: string,
+  isFilled: boolean,
+  required?: boolean,
+}) => (
+  <DefaultFormLabel filled={isFilled} required={required}>
+    {label}
+  </DefaultFormLabel>
+);
+
+class InputStepper extends React.Component<Props, State> {
+  static defaultProps = {
+    theme: defaultTokens,
+  };
+
+  state = {
+    counter: this.props.value || 0,
+  };
+
+  incrementCounter = () => {
+    const { counter } = this.state;
+    const { maxValue = 3, step } = this.props;
+    const newValue = counter + (step || 1);
+    this.setState({
+      counter: newValue > maxValue ? maxValue : newValue,
+    });
+  };
+
+  decrementCounter = () => {
+    const { counter } = this.state;
+    const { minValue = 1, step } = this.props;
+    const newValue = counter - (step || 1);
+
+    this.setState({
+      counter: newValue < minValue ? minValue : newValue,
+    });
+  };
+
+  handleKeyUp = (ev: SyntheticKeyboardEvent<HTMLDivElement>) => {
+    if (ev.keyCode === 38) {
+      this.incrementCounter();
+    }
+  };
+
+  handleKeyDown = (ev: SyntheticKeyboardEvent<HTMLDivElement>) => {
+    if (ev.keyCode === 40) {
+      this.decrementCounter();
+    }
+  };
+
+  render() {
+    const {
+      label,
+      error,
+      onChange,
+      onBlur,
+      onFocus,
+      help,
+      step,
+      disabled,
+      dataTest,
+      size,
+      maxValue = 3,
+      minValue = 1,
+      required,
+    } = this.props;
+    const { counter } = this.state;
+    return (
+      <Field data-test={dataTest}>
+        {label && <FormLabel label={label} isFilled={!!counter} required={required} />}
+        <InputContainer size={size} disabled={disabled} error={error}>
+          {
+            <Prefix disable={counter <= minValue}>
+              <ButtonLink iconLeft={<MinusCircle />} size={size} onClick={this.decrementCounter} />
```

Comment:
I think that disabled should be passed to `ButtonLink` directly.
Something like:
```jsx
<ButtonLink iconLeft={<MinusCircle />} size={size} onClick={this.decrementCounter} disabled={disabled || minValue && counter <= minValue} />
```

Same for the suffix's `ButtonLink`.